docs/src/howto-guides/028_c8y_sevice_monitoring.md

```diff
@@ -0,0 +1,41 @@
+# How to monitor health of service from Cumulocity
+
+The health of a `thin-edge.io` service or any other `service` that is running on the `thin-edge.io` device
+or on the `child` device can be monitored from the Cumulocity by sending the `health-status` message to `Cumulocity` 
+
+Below table gives more information about the `device type`, `thin-edge` topic,  `health status` message format
+and mapping to the `cumulocity` cloud message.
```

Comment:
It's not obvious for a user to guess what he has to do. I would focus on the `tegde/health` topics telling the user when his software/service/child-device has to send a health status and how. In a second step I would tell what is done by thin-edge or more precisely the c8y mapper and how this can be observed.

Reply:
Updated the doc.